core/runtime/binaryen/wasm_memory_impl.cpp

```diff
@@ -90,7 +105,8 @@ namespace kagome::runtime::binaryen {
 
     // Combine with previous chunk if it adjacent
     while (deallocated_.begin() != d_it) {
-      auto d_it_prev = d_it; --d_it_prev;
+      auto d_it_prev = d_it;
+      --d_it_prev;
```

Comment:
Can it not be done somehow like this?
```suggestion
      auto d_it_prev = std::prev(d_it);
```

Reply:
Nice! Really forgot about this way